bundle/manifests/workshop-operator.clusterserviceversion.yaml

```diff
@@ -259,7 +260,7 @@ spec:
                 - --enable-leader-election
                 command:
                 - /manager
-                image: controller:latest
+                image: quay.io/stakater/workshop-operator:v0.0.1
```

Comment:
i have pass image name in `makefile` IMG ?= quay.io/stakater/workshop-operator:v$(VERSION)